lib/stimulus_reflex/reflex.rb

```diff
@@ -103,4 +105,8 @@ def halted?
   def default_reflex
     # noop default reflex to force page reloads
   end
+
+  def params
+    @_params ||= ActionController::Parameters.new(request.parameters)
+  end
```

Comment:
I'd prefer to see this implemented as:

```ruby
delegate :params, to: :request
```

Reply:
I'm down to do that. 

Where would we get the `ActionController::Parameters` instance, then? My understanding of ActionController is that a request isn't responsible for the instantiation of strong params. I believe it's a module that's included in `ActionController::Base` that reads the params out of the request. The method here is actually from that module.

Reply:
I'm expecting Rails internals to set that up for us. What happens when if we delegate? Is `params` and instance of `ActionController::Parameters`?

Reply:
I finally caught up to you on this and can see that you simply implemented what the controller is doing. This makes sense to me now.